src/features/environment.js

```diff
@@ -167,6 +176,16 @@ export const EnvironmentMixin = (ModelViewerElement) => {
       this[$needsRender]();
     }
 
+    [$updateShadow]() {
+      const {shadowStrength} = this;
+      const shadowStrengthIsNumber =
```

Comment:
Doesn't lit-element handle some of this?

Reply:
Lit won't stop the value from being `NaN` when a non-number value is set in the attribute. And, any value may be assigned to the property, Lit does not do any filtering or validation there.